easyfsl/samplers/task_sampler.py

```diff
@@ -80,6 +80,33 @@ def episodic_collate_fn(
                 - their labels,
                 - the dataset class ids of the class sampled in the episode
         """
+        #the input_data should be a list
```

Comment:
(!) I don't think we need to check that `input_data` is a list as it is a construction of `torch.utils.data.DataLoader` which we can trust. Plus, if it's not a list, it will break very fast so it's not a huge risk.